dbt-metricflow/dbt_metricflow/cli/dbt_connectors/adapter_backed_client.py

```diff
@@ -177,14 +183,15 @@ def execute(
             )
         start = time.time()
         request_id = SqlRequestId(f"mf_rid__{random_id()}")
-        logger.info(AdapterBackedSqlClient._format_run_query_log_message(stmt, sql_bind_parameters))
+        logger.info(LazyFormat("Running query", statement=stmt, param_dict=sql_bind_parameters.param_dict))
```

Comment:
Can we differentiate the log message for `execute` from `query`? Makes it harder to figure out where a log is coming from if the message is logged in multiple places!

Reply:
Updated.